### PR TITLE
Allow sury repo for PHP >= 7.1.

### PR DESCRIPTION
**Pull request by Hexta**
#### Pull Request (PR) description
Allow sury repo for PHP >= 7.1.

### CODE REVIEW

**Comment by Hexta** [comment]
Could anybody please  take a look at the PR?

File: manifests/repo/debian.pp [hunk 1]
```diff
@@ -67,7 +67,13 @@
     }
   }
 
-  if ($sury and $php::globals::php_version in ['5.6','7.1','7.2'] ) {
```
Comment (c33s) [inline manifests/repo/debian.pp]: would be better to simply add the supported php version of sury in the array, only this way non valid php versions could be catched.
with your change a php version like `8.0` would be allowed

Comment (Hexta) [reply manifests/repo/debian.pp]: Do you really think that hardcoded list of supported php versions is a good idea?

Comment (c33s) [reply manifests/repo/debian.pp]: not really, i prefer to have them configureable via hiera with sane default values directly set in this module with hiera. i would prefer that for many hardcoded values (like the repo key id) but without a bigger refactoring this is not possible.

to simply remove the version check feels wrong to me.

**Comment by bastelfreak** [comment]
@Hexta please rebase against our latest master branch.

**Pull request closed without merging**